uint8 left_vel
uint8 right_vel
uint8 auger
uint8 dump
uint8 reserved
uint8 reserved1
uint8 reserved8
uint8 verification
